packages/vscode-extension/src/dependency/DependencyManager.ts

```diff
@@ -315,6 +320,26 @@ export class DependencyManager implements Disposable {
     return installed;
   }
 
+  public async checkExpoRouterInstalled() {
+    const status = checkMinDependencyVersionInstalled(
```

Comment:
this function alone does not give as information if project uses expo router or not